<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtensionSystemConfig</class>
 <widget class="QWidget" name="ExtensionSystemConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Plugins</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLoadedPlugins">
      <attribute name="title">
       <string>Loaded Plugins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QWidget" name="widgetPluginListHolder" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnPluginDetails">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Details</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblStatusMessage">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPaths">
      <attribute name="title">
       <string>Plugin Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelPluginPaths">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>18</height>
          </size>
         </property>
         <property name="text">
          <string>Plugins loaded from 1 path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listPluginPaths"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
